src/logger.js

```diff
@@ -0,0 +1,7 @@
+import bunyan from 'bunyan';
+
+export default bunyan.createLogger({
+  name: 'AddonValidatorJS',
+  stream: process.stdout,
+  level: process.env.LOG_LEVEL || 'fatal',
```

Comment:
Could we have a test for this so a log level other than an acceptable one either errors or defaults to fatal? I like an error better, just in case someone types a log level in that's wrong and gets stumped by the output.